front/app/containers/Admin/projects/all/Lists/NonSortableFolderRow.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { isNilOrError } from 'utils/helperUtils';
+
+// style
+import useAdminPublications, {
+  IAdminPublicationContent,
+} from 'hooks/useAdminPublications';
+import ProjectFolderRow from '../../projectFolders/components/ProjectFolderRow';
+import { PublicationStatus } from 'services/projects';
+import { Row } from 'components/admin/ResourceList';
+import useAuthUser from 'hooks/useAuthUser';
+import FolderChildProjects from './FolderChildProjects';
+interface Props {
+  key: number;
+  id: string;
+  isLastItem: boolean;
+  publication: IAdminPublicationContent;
+}
+
+const publicationStatuses: PublicationStatus[] = [
+  'draft',
+  'published',
+  'archived',
+];
+
+const SortableFolderRow = ({ key, id, isLastItem, publication }: Props) => {
+  const authUser = useAuthUser();
+  const [folderOpen, setFolderOpen] = useState(true);
+
+  const toggleFolder = () => {
+    setFolderOpen((folderOpen) => !folderOpen);
```

Comment:
Minor: Any chance we could only toggle this when a folder has projects in it. At the moment, this will cause a rerender